Assets/Plugins/Source/Editor/Configs/LibraryBuildConfig.cs

```diff
@@ -21,10 +21,19 @@
  */
 namespace PlayEveryWare.EpicOnlineServices.Editor.Config
 {
+    using System;
+
+    [Serializable]
```

Comment:
I noticed this EditorConfig is serializable, but not `PackagingConfig`. Should all EditorConfig files be marked with this?

Reply:
For the sake of clarity, it should be. Functionally, the `[Serializable]` attribute is actually inherited by extending the `EditorConfig` class, which itself has that attribute. For the sake of consistency however, I have added it here.

Reply:
I agree with Paul.